spec/lib/local-links-manager/import/local_authorities_importer_spec.rb

```diff
@@ -31,7 +31,7 @@ def fixture_file(file)
 
           expect(la.name).to eq("Aberdeen City Council")
           expect(la.snac).to eq("00QA")
-          expect(la.tier_id).to eq(Tier.unitary)
+          expect(la.tier).to eq('unitary')
```

Comment:
The changes in this file should probably be rolled into the last commit as that introduced the `tier` method for LocalAuthority